### PR TITLE
a new root page and test

### PR DESCRIPTION
**Pull request by lei-appfolio**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ group :development do
 end
 
 group :production do
-  gem 'pg'
+  # gem 'pg'
```
Comment (lsq921224) [inline Gemfile]: This should be in a separate commit because it's not part of adding the new page

File: test/controllers/application_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class ApplicationControllerTest < ActionDispatch::IntegrationTest
+  test 'name should be World' do
+    obj = ApplicationController.new
+    obj.home
+    assert obj.instance_variable_get(:@name) == 'World'
```
Comment (lsq921224) [inline test/controllers/application_controller_test.rb]: This is ok, but as a controller test, it's not testing the view at all. For example, the `Hello` text was not tested. It's better to use `assert_select` here and test if the controller rendered Hello World

File: Gemfile (hunk 1 above)
Comment (lei-appfolio) [reply Gemfile]: Got it!

File: test/controllers/application_controller_test.rb (hunk 2 above)
Comment (lei-appfolio) [reply test/controllers/application_controller_test.rb]: Got it!

File: test/controllers/application_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  test 'test view content' do
+    get '/'
+    assert_equal 200, status
```
Comment (lsq921224) [inline test/controllers/application_controller_test.rb]: `assert_response :ok` is more standard way of testing response

Comment (lei-appfolio) [reply test/controllers/application_controller_test.rb]: Got it!
